### ISSUE TITLE
[MSFT-120] Challenges page: Screen reader doesn't announce the type of element for the 'compte ' and 'community' controls.

### ISSUE THREAD

**Issue by veshu** [importtest/importtest#120]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure screen reader is on. 3. Press down arrow to read the 'compte' and 'community' controls. 3. Notice that the screen reader doesn't announce these as either buttons or links.

**Expected Result**
Compete and community controls should be announced as links since they move focus to a different page.

**Actual Result**
Screen reader doesn't announce the type of element for the 'compete' and 'community' control.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 4. Robust
**WCAG Level**: A
**WCAG Success Criterion**: 4.1.2 Name, Role, Value
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2031
